pkg/cgroups/utils.go

```diff
@@ -150,7 +150,21 @@ func createCgroupv2Path(path string) (deferredError error) {
 		// PIDs if there are already enabled controllers.
 		if i < len(elements[3:])-1 {
 			if err := ioutil.WriteFile(filepath.Join(current, "cgroup.subtree_control"), res, 0o755); err != nil {
-				return err
+				if os.IsNotExist(err) {
+					if _, err := os.Stat(current); err != nil {
+						// one of the parent directories is not created
+						err = os.Mkdir(current, 0o755)
```

Comment:
I'm super-ignorant about cgroups, but isn't this exactly what line 136 just tried to do?

Reply:
yeah, I think there might be some sort of loop race here with mkdir and writefile. I will look for more of an elegant solution if this does "fix" it, for now just trying to pinpoint @edsantiago